apps/tup-cms/src/apps/user_news/cms_plugins.py

```diff
@@ -32,3 +32,24 @@ def render(self, context, instance, placeholder):
             'urls': urls
         })
         return context
+
+
+@plugin_pool.register_plugin
+class UserUpdatesPlugin(CMSPluginBase):
+    module = 'TUP CMS'
+    name = _('User Updates')
```

Comment:
[Background discussion.](https://github.com/TACC/tup-ui/pull/438#issuecomment-1982034249)

**4. ~~Rename plugins.~~**
> @R-Tomas-Gonzalez: The [User News List] plugin is already being used with it's name and therefore the template is looking for this plugin name. If we remove the plugin, we must delete first before deploying this change. […] We can make this change ("Full User News List" → "User Updates").

Well, crap. Thanks. Okay, then your original name is better. Please revert `e4ced86`.

I will consider whether to make a new ticket to rename user-facing name of both plugins (a semantic courtesy for the CMS admin).[^1]

[^1]: I think https://stackoverflow.com/questions/77105970/how-to-rename-a-djangocmss-plugin and https://gist.github.com/yakky/31466588da712e25e1f2c200d56d3776 could help.

Reply:
Reverted.